src/middleware/testingOnlyMiddleware.ts

```diff
@@ -0,0 +1,20 @@
+// import { Request, Response } from 'express';
+
+export const isTestingOrCI = () => {
+  const isLocal = process.env.NODE_ENV === 'development'
+    || process.env.NODE_ENV === 'test';
+  const isCI = process.env.CI !== undefined
+    && process.env.CI !== null;
+
+  return isLocal || isCI;
```

Comment:
This might be fine but it is a little weird that process.env.NODE_ENV = "production" and process.env.CI = "false" (or false) would return true from this function. I'm assuming that the process.env.CI value comes from circleCI and is unpredictable?